From 73ac9a1227bd21750f3f12cc37a72596f50d83cf Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 7 Aug 2016 12:17:05 +0300
Subject: [PATCH 0390/1345] pp3: fw-version print without carriage-return

commit  a4a8220d7a146572a5fe5d2c6c43059f3981ebe6 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Fix print of FW version
Incorrect variant:
   Driver version: NSS:16.07.0
   FW version:     NSS:16.07.1
   .3
Correct variants:
 a)."debug_d" extension is not null:
   Driver version: NSS:16.07.0
   FW version:     NSS:16.07.1.3
 b)."debug_d" extension is null:
   Driver version: NSS:16.07.0
   FW version:     NSS:16.07.1

Change-Id: I46d5904688f541bf966f7b97498cab8f8a9172aa
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31761
Reviewed-by: Dovrat Zifroni <dovrat@marvell.com>
Tested-by: Dovrat Zifroni <dovrat@marvell.com>
Reviewed-by: Dmitri Epshtein <dima@marvell.com>
Tested-by: Dmitri Epshtein <dima@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/pp3/platform/mv_pp3.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3.c b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3.c
index 0be310c..b2b7051 100644
--- a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3.c
+++ b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3.c
@@ -690,7 +690,7 @@ int mv_pp3_shared_start(struct mv_pp3 *priv)
 		pr_cont(".%d\n", drv_ver->debug_d);
 
 	memcpy(version_name, fw_ver.name, ver_name_size);
-	pr_info("FW version:     %s:%02d.%02d.%d\n",
+	pr_info("FW version:     %s:%02d.%02d.%d",
 		version_name, fw_ver.major_x, fw_ver.minor_y, fw_ver.local_z);
 	if (fw_ver.debug_d)
 		pr_cont(".%d\n", fw_ver.debug_d);
-- 
1.7.9.5

